pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "nautobot-ssot"
-version = "2.0.0"
+version = "2.0.0-beta.2"
```

Comment:
Shouldn't this be beta 1 of SSoT?

Reply:
I was doing this to match Nautobot but I'll change it to beta 1. Will also add a compatibility matrix to documentation so it's clear what version is supported for which Nautobot version.